services/web/server/src/simcore_service_webserver/projects/_handlers_project_nodes.py

```diff
@@ -364,28 +352,20 @@ async def replace_node_resources(request: web.Request) -> web.Response:
     except json.JSONDecodeError as exc:
         raise web.HTTPBadRequest(reason=f"Invalid request body: {exc}") from exc
 
-    try:
-        # ensure the project exists
-        _project = await projects_api.get_project_for_user(
-            request.app,
-            project_uuid=f"{path_params.project_id}",
-            user_id=req_ctx.user_id,
-        )
-        raise web.HTTPNotImplemented(reason="Not yet implemented!")
-        # new_node_resources = await projects_api.set_project_node_resources(
-        #     request.app, project=project, node_id=node_id
-        # )
-
-        # return web.json_response({"data": new_node_resources}, dumps=json_dumps)
+    # ensure the project exists
+    _project = await projects_api.get_project_for_user(
+        request.app,
+        project_uuid=f"{path_params.project_id}",
+        user_id=req_ctx.user_id,
+    )
+    raise web.HTTPNotImplemented(reason="Not yet implemented!")
+    # NOTE: SAN? This your code?
+    #
+    # new_node_resources = await projects_api.set_project_node_resources(
+    #     request.app, project=project, node_id=node_id
+    # )
```

Comment:
@sanderegg @pcrespov have you guys figured out what to do with this? if possible remove it or add a comment to why you want to keep it

Reply:
We will keep it for the moment.